CONTRIBUTING.md

```diff
@@ -71,6 +71,28 @@ To specify imagery taken sometime in 2019, use `"start_date": "2019"`,
 Implementations *must not* round down the end date (e.g. consider `2013` the same as the
 start of `2013`. Note that this is the opposite of what we did before, and layers before 2015 could have overly wide date ranges.
 
+
+##### Categories
+
+Sources should be classified into one of the following categories of different types of layers:
+
+* `photo` – For satellite and aerial imagery.
+* `historicphoto` – Same as above, but for `photo` layers which are either quite old[^1] or when there exists a more recent `photo` source for the same region from the same imagery provider.
+* `map` – A map which is not mostly based on OSM data.
+* `historicmap` – Same as above, but for `map` layers which are either quite old[^1] or when there exists a more recent version of the map for the same region from the same provider.
+* `elevation` – A map focused on displaying terrain cartography, such as elevation contour lines or shaded relief maps.
+* `osmbasedmap` – A map which is mostly based on OSM data.
+* `qa` – A map or overlay with the main intent 
```

Comment:
.. with the main intent ??? the end of the sentence is missing here.

Reply:
oops, the rest of the sentence was missing (and the description for `other`), sorry for the oversight. It should be fixed now.